packages/core/src/browser/core-preferences.ts

```diff
@@ -38,13 +38,20 @@ export const corePreferenceSchema: PreferenceSchema = {
             ],
             default: 'ifRequired',
             description: 'When to confirm before closing the application window.',
+        },
+        'workbench.commands.history': {
```

Comment:
Is it aligned with VS Code? could you point me to definition in VS Code repo

Reply:
Do we want to align completely with VSCode for the preference name, and description?
https://github.com/Microsoft/vscode/blob/473af338e1bd9ad4d9853933da1cd9d5d9e07dc9/src/vs/workbench/browser/workbench.contribution.ts#L128

Reply:
yes, we want, Theia should be compatible with VS Code preferences

Reply:
Sounds good :) I'll make the changes and open a CQ for the copied preference